ios/PurchasesHybridCommon/PurchasesHybridCommonTests/Mocks/MockPurchases.swift

```diff
@@ -0,0 +1,491 @@
+//
+// Created by Andrés Boedo on 4/7/21.
+// Copyright (c) 2021 RevenueCat. All rights reserved.
+//
+
+import Foundation
+import Purchases
+
+class MockPurchases: Purchases {
```

Comment:
big mock here. Mostly autogenerated, but I fixed some of the names by hand. Only a part of it is used, but I figured it's safer to do a full mock and not risk calling real methods. 
Also, this way we already have stuff set up for future tests.